### ISSUE TITLE
Create package for course evaluations

### ISSUE THREAD

**Issue by joel-coffman** [joel-coffman/latex-usafa.edu#39]
Create a standardized report for course evaluations ([example](https://drive.google.com/file/d/1lsiXn6VQU3L7mvs0pFSMQqfLGSu14Abm/view?usp=sharing)) that standardizes report elements (e.g., bar graphs).

**Comment by joel-coffman** [comment]
Python code snippet to calculate scores for questions:

~~~ python
# repeat the following code for each question
responses = []  # TODO: Insert count of each response choice
values = range(1, len(responses) + 1)
scores = [values[i] for i, x in enumerate(responses) for y in range(x)]
tuple([round(x, 3) for x in [numpy.mean(scores), numpy.std(scores)]])
~~~

**Comment by joel-coffman** [comment]
Addressed by #49 (ffd7d12).
